src/AuthMSAL/AuthMSAL.js

```diff
@@ -220,6 +235,10 @@ function getUserId() {
   return undefined;
 }
 
+function getUserRoles() {
+  return authData.roles;
```

Comment:
Maybe we should define a default value for `authData.roles` in case we try to read it before `_extractRolesFromAccessToken` has been called (c.f. lines 22-27 for init).

Reply:
Add empty array as default value